scripts/compareAPIs.js

```diff
@@ -22,7 +22,13 @@ async function compare() {
   let branchDir = path.join(__dirname, '..', 'dist', 'branch-api');
   let publishedDir = path.join(__dirname, '..', 'dist', 'published-api');
   if (!(fs.existsSync(branchDir) && fs.existsSync(publishedDir))) {
-    console.log('one of the api directories does not exist');
+    if (!fs.existsSync(publishedDir && fs.existsSync(branchDir))) {
+      console.log('not showing private package', branchDir);
+    } else if (fs.existsSync(publishedDir && !fs.existsSync(branchDir))) {
```

Comment:
Are we supposed to call `existsSync` like that? Or is the closing parenthesis off?

Reply:
ack! good catch! good thing it's not a common code path